examples/mpi/cntk-mpi.json

```diff
@@ -1,6 +1,6 @@
 {
   "jobName": "cntk-mpi",
-  "image": "paiexample/pai.example.cntk-mpi",
+  "image": "openpai/pai.example.cntk-mpi",
 
   // prepare cmudict corpus in CNTK format https://git.io/vbT5A and upload to hdfs
   "dataDir": "$PAI_DEFAULT_FS_URI/path/cntk-mpi/data",
```

Comment:
What about we remove comments from the all example jsons so that users can use it directly?

Reply:
Not all of the examples can be submitted directly, we can not put large dataset in images.
[Here](https://github.com/Microsoft/pai/tree/master/examples#off-the-shelf-examples) lists off-the-shelf examples.

Reply:
I suggest we support yaml file in the future. single line comment is supported in yaml

---
In reply to: [197685206](https://github.com/Microsoft/pai/pull/763#discussion_r197685206) [](ancestors = 197685206)

Reply:
ok